src/components/tooltip-popup/styles.scss

```diff
@@ -1,10 +1,12 @@
 @use '~@zero-tech/zui/styles/theme' as theme;
 @use '~@zero-tech/zui/styles/colors' as colors;
+@import '../../glass';
 
 .tooltip-popup {
-  display: block;
+  position: fixed;
```

Comment:
Does this break the side and align settings? `data-side='right'][data-align='end'`

Reply:
currently it doesn't, but i also haven't tested a lot of cases. just for the one we need to display. (arrow pointed left, data pointed right)